### PR TITLE
Hyprland on NixOS: add section detailing how to fix problems with themes not working

### PR DESCRIPTION
**Pull request by Schweber**
I found out about this comprehensive and easy solution yesterday and it solved all my problems with themes not loading since switching to wayland/hyprland. I think that this is useful for others and would save them time and hassle trying to fix such a problem.

### CODE REVIEW

**Review by fufexan — CHANGES_REQUESTED** [review]
Should be in the Hyprland on Home-Manager page.

File: pages/Nix/Hyprland on Home Manager.md [hunk 1]
```diff
@@ -126,3 +126,43 @@ wayland.windowManager.hyprland.plugins = [
 
 For examples on how to build hyprland plugins using nix see the
 [official plugins](https://github.com/hyprwm/hyprland-plugins).
+
+## Fixing problems with themes
+
+If your themes for mouse cursor, icons or windows don't load correctly, try setting them with `home-manager`. `home-manager` enables a bunch of compatibility options that should make the themes load in all situations.
```
Comment (fufexan) [inline pages/Nix/Hyprland on Home Manager.md:132]:
```suggestion
If your themes for mouse cursor, icons or windows don't load correctly, try setting them with `home.pointerCursor` and `gtk.theme`, which enable a bunch of compatibility options that should make the themes load in all situations.
```

**Comment by Schweber** [comment]
I moved the detailed section to Hyprland on Home Manager and left a small reference in Hyprland on NixOS so people that are not already using home-manager notice it. Otherwise they might not look into Hyprland on Home Manager at all.

Is that ok?

(it's my first PR so apologies, i'm still fighting with Github and the whole process)

**Comment by Schweber** [comment]
I moved the detailed section to Hyprland on Home Manager and left a small reference in Hyprland on NixOS so people that are not already using home-manager notice it. Otherwise they might not look into Hyprland on Home Manager at all.

Is that ok?

(it's my first PR so apologies, i'm still fighting with Github and the whole process)

File: pages/Nix/Hyprland on Home Manager.md (hunk 1 above)
Comment (fufexan) [reply pages/Nix/Hyprland on Home Manager.md:132]: You marked this as resolved but didn't apply the suggestion.

Comment (Schweber) [reply pages/Nix/Hyprland on Home Manager.md]: Sorry, i've commited your suggestion now. (a47cd03)

Please tell me if the other changes are ok or should be removed

File: pages/Nix/Hyprland on NixOS.md [hunk 2]
```diff
@@ -94,3 +94,7 @@ in {
 
 {{< /tab >}}
 {{< /tabs >}}
+
+## Fixing problems with themes
+
+If your themes for mouse cursor, icons or windows don't load correctly, see the relevant section in [Hyprland on Home Manager](../Hyprland on Home Manager).
```
Comment (fufexan) [inline pages/Nix/Hyprland on NixOS.md:100]:
```suggestion
If your themes for mouse cursor, icons or windows don't load correctly, see the relevant section in [Hyprland on Home Manager](./Hyprland-on-Home-Manager).
```

**Review by fufexan — APPROVED** [review]
LGTM! Thanks for the contribution.

**Comment by Schweber** [comment]
Thank you for your patience ;-)

**Pull request merged**